src/AdminConsole/Pages/Organization/Admins.cshtml.cs

```diff
@@ -92,28 +94,39 @@ public async Task<IActionResult> OnPostDelete(string userId)
     public async Task<IActionResult> OnPostInvite(InviteForm form)
     {
         CanInviteAdmin = await _dataService.CanInviteAdminAsync();
+
         if (CanInviteAdmin is false)
         {
             ModelState.AddModelError("error", "You need to upgrade to a paid organization to invite more admins.");
             return await OnGet();
         }
 
+        var org = await _dataService.GetOrganizationAsync();
+        var existingInvites = await _invitationService.GetInvitesAsync(org.Id);
+
+        if (existingInvites.Count >= 10)
+        {
+            ModelState.AddModelError("error", "You can only have 10 pending invites at a time.");
+            return await OnGet();
+        }
+
+        if (existingInvites.Any(x => string.Equals(x.ToEmail, form.Email, StringComparison.CurrentCultureIgnoreCase)))
```

Comment:
I don't think non-ascii characters can be used in emails, can they?

```suggestion
        if (existingInvites.Any(x => string.Equals(x.ToEmail, form.Email, StringComparison.OrdinalIgnoreCase)))
```